src/Library/demos/Frame/main.blp

```diff
@@ -14,20 +14,144 @@ Adw.StatusPage {
       halign: center;
       spacing: 18;
       margin-bottom: 18;
+      homogeneous: false;
+
+      Picture without_frame {
+        width-request: 180;
+        height-request: 180;
+      }
+
+      Separator {
+        margin-start: 6;
+        margin-end: 6;
+      }
+
+      Frame {
+        Picture with_frame {
+          width-request: 180;
+          height-request: 180;
+        }
+      }
+    }
+
+    Box {
+      orientation: horizontal;
       homogeneous: true;
+      margin-bottom: 24;
+
+      Label {
+        label: _("Without Frame");
+        styles ["title-4"]
+      }
 
       Label {
-        wrap: true;
+        label: _("With Frame");
+        styles ["title-4"]
+      }
+    }
+
+    Separator {
+      margin-bottom: 24;
+    }
+
+    Label {
+      label: _("Without Frame");
+      margin-bottom: 12;
+      halign: start;
+      styles ["title-4"]
+    }
+
+    Box {
+      margin-bottom: 18;
+      halign: center;
+      styles ["toolbar"]
+
+      Button {
+        icon-name: "plus";
+      }
+
+      Button {
+        icon-name: "minus";
+      }
+
+      Button {
+        icon-name: "bell";
+      }
+    }
+
+    Label {
+      halign: start;
+      label: _("With Frame");
+      margin-bottom: 12;
+      styles ["title-4"]
+    }
+
+
+    Frame {
```

Comment:
This one would be good to demonstrate the `label` property or `label-widget` property. The simplest thing would be to just set `Gtk.Frame:label` to something like `_("Label that handles Accessible setup")`.

It's not commonly used, but has the benefit of setting up accessible roles and relationships automatically.